### DISCUSSION TITLE
bad user-agents

### DISCUSSION THREAD

**Discussion by glikis** [mitchellkrogza/nginx-ultimate-bad-bot-blocker#574] [General]
Hello,

I was wondering whether you maintain the bad user-agents/bots in their full form as well.

Thank you!

**Comment by mitchellkrogza** [comment]
> Hello,
> 
> I was wondering whether you maintain the bad user-agents/bots in their full form as well.
> 
> Thank you!

Hi @glikis do you mean the full agent string like BOT/1.0 BOT/1.2 ???
